components/brave_wallet/browser/eth_tx_controller.cc

```diff
@@ -222,8 +222,9 @@ void EthTxController::ApproveTransaction(const std::string& tx_meta_id,
   }
 
   if (!meta->last_gas_price) {
+    auto from = EthAddress(meta->from);
     nonce_tracker_->GetNextNonce(
-        meta->from,
+        from,
```

Comment:
since we don't use it else where, we can directly pass `EthAddress(meta->from)` without creating an object to hold it